runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowWorkProgressUpdater.java

```diff
@@ -58,7 +66,6 @@ public DataflowWorkProgressUpdater(
    * virtual time for testing, and the {@link Clock} parameter is used to inject a mock clock that
    * provides virtual time.
    */
-  @VisibleForTesting
   DataflowWorkProgressUpdater(
```

Comment:
Why did you remove the visiblefortesting annotation?

Reply:
Must have been a bad merge, added back.